torchvision/prototype/transforms/functional/_type_conversion.py

```diff
@@ -27,20 +27,25 @@ def label_to_one_hot(label: torch.Tensor, *, num_categories: int) -> torch.Tenso
 
 
 def to_image_tensor(image: Union[torch.Tensor, PIL.Image.Image, np.ndarray], copy: bool = False) -> torch.Tensor:
+    if isinstance(image, np.ndarray):
+        image = torch.from_numpy(image)
```

Comment:
This makes the kernel non-jit scriptable.

Reply:
was this kernel scriptable before ?

Reply:
Hmm I guess not because of the invocation of `pil_to_tensor`.